### PR TITLE
Update device-control-removable-storage-access-control.md

### PR DESCRIPTION
**Pull request by msbemba**
updated navigation path for device control policy under attack surface reduction

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6421

File: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md [hunk 1]
```diff
@@ -200,7 +200,7 @@ Microsoft Endpoint Manager admin center (<https://endpoint.microsoft.com/>) \> *
 
 ## Deploying and managing policy by using Intune user interface
 
-This capability (in Microsoft Endpoint Manager admin center (<https://endpoint.microsoft.com/>) \> Devices \> Configuration profiles \> Create profile \> Platform: Windows 10 and later & Profile: Device Control) is not yet available.
+This capability is availabile at  Microsoft Endpoint Manager admin center (<https://endpoint.microsoft.com/>) \> Endpoint Security \> Attack Surface Reduction \> Create Policy \> Platform: Windows 10 and later & Profile: Device Control)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md]:
```suggestion
This capability is availabile at Microsoft Endpoint Manager admin center (<https://endpoint.microsoft.com/>) \> Endpoint Security \> Attack Surface Reduction \> Create Policy \> Platform: Windows 10 and later & Profile: Device Control).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**